RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py

```diff
@@ -67,6 +67,11 @@
     Pt4050_Loose = cms.vdouble(-0.19, 0.22, -0.13, -0.03)
 )
 
+##########################################################
+## Working points for the 106X UL18 training
+###########################################################
+full_106x_UL18_chs_wp = full_106x_UL17_chs_wp.clone()
```

Comment:
updates here remind me of #30776 

I propose to do this cleanup now

Reply:
I don't know what do you mean by cleanup? Just delete old stuff?

Reply:
yes, please.
this is in line with https://github.com/cms-sw/cmssw/pull/30773#discussion_r456513115
`We will make sure to clean up this file on the next iteration.`
which in turn was the reason to open #30776

Reply:
Alright. I will work on it. Thanks.